CHANGES.rst

```diff
@@ -28,7 +28,10 @@ New Features:
 
 - Added `flip('...')` function support in `select` query to `postgresql` and `sql` backends (`#1052`_).
 
+- Added OpenAPI Schema manifest (`#1211`_)
+
   .. _#1052: https://github.com/atviriduomenys/spinta/issues/1052
+  .. _#1211: https://github.com/atviriduomenys/spinta/issues/1211
```

Comment:
This needs to go to the 2nd version of spinta (1st is feature-freezed), so it needs to be moved higher, under "0.2dev2 (unreleased)"

Reply:
Perkelta į nurodytą vietą ir sulieta su pagrindine šaka (galbūt ne tam pačiam PR'e).